src/test/java/org/kafkahq/repositories/UserRepositoryTest.java

```diff
@@ -0,0 +1,95 @@
+package org.kafkahq.repositories;
+
+import org.apache.kafka.common.resource.ResourceType;
+import org.junit.jupiter.api.Test;
+import org.kafkahq.AbstractTest;
+import org.kafkahq.KafkaTestCluster;
+import org.kafkahq.models.User;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UserRepositoryTest extends AbstractTest {
+
+    @Inject
+    private UserRepository userRepository;
+
+    @Test
+    public void findAll() {
+        assertEquals(2,
+                userRepository.findAll(KafkaTestCluster.CLUSTER_ID, Optional.empty()).size());
+    }
+
+    @Test
+    public void findAll_with_filter() {
```

Comment:
just case here and bottom :p 
I hate python, why `_` that remember me python :laughing:

Reply:
omg, my brain just randomized tha naming on these ones 😆